spyder/plugins/completion/languageserver/transport/tcp/producer.py

```diff
@@ -70,6 +70,10 @@ def transport_send(self, content_length, body):
             self.socket.send(content_length)
             self.socket.send(body)
         except (BrokenPipeError, ConnectionError) as e:
+            # This avoids a total freeze at startup
+            # when we're trying to connect to a TCP
+            # (but not LSP) server that rejects our
```

Comment:
`(but not LSP). I don't understand that phrase @ccordoba12. What do you mean?

Reply:
It means any TCP server (for example an http server). The problem is we don't have a way to tell if a server is LSP because the LSP protocol doesn't provide the means for that.

Reply:
We could paraphrase it to refer to a TCP socket instead of a server

Reply:
Ok, will do.